frontend/express/public/stylesheets/vue/clyvue.scss

```diff
@@ -23,6 +23,26 @@
     }
 }
 
+.cly-vue-color-tag {
```

Comment:
this should be added to _helpers.scss in frontend/express/public/stylesheets/styles/blocks/_helpers.scss